README.md

```diff
@@ -52,6 +52,7 @@ import { ReactTinyLink } from 'react-tiny-link'
 | **defaultMedia** | Default value to provide the media for failure cases      | string                          | N.A                                            | `false`  |
 | **onError**      | Error callback on when the url failed to fetch            | onError(`error:Error`)          | N.A                                            | `false`  |
 | **onSuccess**    | Success callback on when the url was fetched successfully | onSuccess(`data:IResponseData`) | N.A                                            | `false`  |
+| **onlySecure**    | Load only secure ( `https://` ) resources. If no secure resource was found, then don't render the `<img>` and `<video>` element  | boolean | default(`false`)                                            | `false`  |
```

Comment:
Shall we name it to `loadSecureUrls` ?

Reply:
or `loadOnlySecureUrls` maybe?

Reply:
renamed `onlySecure` props to `loadSecureUrls`

Reply:
Hi @sndsabin I think i typed wrong prop name actually it should be singular `loadSecureUrl`